tests/conftest.py

```diff
@@ -238,3 +239,38 @@ def unprivileged_client(
 
         else:
             yield admin_client
+
+
+@pytest.fixture(scope="session")
+def dsc_resource(admin_client: DynamicClient, name: str = "default-dsc"):
+    for dsc in DataScienceCluster.get(dyn_client=admin_client, name=name):
+        yield dsc
+
+
+def get_dsc_component_state(dsc_resource: DataScienceCluster, component_name: str) -> str:
+    return dsc_resource.instance.spec.components[component_name].managementState
+
+
+@pytest.fixture(scope="module")
+def set_dsc_component_state(
```

Comment:
a fixture name should be a noun and not a verb; i.e describes the result.
e.g updated_dsc_component_state`